app/src/main/java/org/fossasia/openevent/app/core/main/MainActivity.java

```diff
@@ -195,7 +210,9 @@ private void loadFragment(int navItemId) {
                 fragment = EventDashboardFragment.newInstance(eventId);
                 break;
         }
-        fragmentManager.beginTransaction().replace(R.id.fragment_container, fragment).commit();
+
+        ft.add(R.id.fragment_container, fragment).addToBackStack("from dashboard to fragment").commit();
```

Comment:
Extract the tag and use simpler string

Reply:
It's just an optional name for the back state. So can I use `null`?